commercetools-models/src/test/java/io/sphere/sdk/discountcodes/commands/DiscountCodeUpdateCommandIntegrationTest.java

```diff
@@ -75,6 +75,18 @@ public void setMaxApplicationsPerCustomer() throws Exception {
         });
     }
 
+
+    @Test
+    public void setGroups() throws Exception {
```

Comment:
Naming this test `changeGroups` would be more consistent with our other tests which are always named like the update action under test. But I saw that you already discussed the naming on our documentation PR...

Reply:
👍  good catch